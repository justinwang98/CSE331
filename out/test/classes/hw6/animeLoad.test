# Test the parser on a graph that contains several nodes
# with two edges each.

LoadGraph anime anime.tsv
ListChildren anime Naruto