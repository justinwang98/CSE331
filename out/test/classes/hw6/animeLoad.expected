# Test the parser on a graph that contains several nodes
# with two edges each.

loaded graph anime
the children of Naruto in anime are: Asuna(Naruto) Asuna(SAO) Itachi(Naruto) Kirito(OnePiece) Kirito(SAO) Luffy(Naruto) Luffy(OnePiece) Sanji(OnePiece) Sasuke(Naruto) Zoro(OnePiece)
